src/pool.h

```diff
@@ -28,19 +28,11 @@ typedef struct git_pool_page git_pool_page;
  * For examples of how to set up a `git_pool` see `git_pool_init`.
  */
 typedef struct {
-	git_pool_page *open; /* pages with space left */
-	git_pool_page *full; /* pages with no space left */
-	void *free_list;     /* optional: list of freed blocks */
+	git_pool_page *pages; /* pages with space left */
```

Comment:
The comment is now inaccurate since we store a single list, so this is just the count of pages we've allocated.